app/models/collab.rb

```diff
@@ -1,4 +1,5 @@
 class Collab < ApplicationRecord
   validates :name, presence: true, uniqueness: true
-  has_many :users_collab, dependent: :destroy
+  has_many :users_collabs, dependent: :destroy
+  has_many :users, through: :users_collabs, dependent: :destroy
```

Comment:
not sure to understand what the second dependent destroy should do

Reply:
I'm not very clear about dependant: :destroy  in general but if you delete a user it should delete the collab also

Reply:
Yes but then the dependent: :destroy should be on the side of the user model no? 
That line I have the impression it's saying : when you delete a collab, delete his users

Reply:
yes, you are right, it's on collab model

Reply:
Yes, I would remove the line 4

Reply:
no the line 4 shouldn't be removed, just the dependent destroy